docs/versioned/eventing/faq/README.md

```diff
@@ -1,13 +0,0 @@
----
-audience: developer
-components:
-  - eventing
-function: reference
----
-
-# Eventing FAQs
-
-## What is a Sugar Controller?
-
-The Sugar Controller is an optional controller which will automatically create and clean up brokers based on the creation/deletion of namespaces and triggers.
-For more information on how to enable and configure the sugar controller, read the [sugar controller documentation](../sugar/README.md).
```

Comment:
I'd prefer not to remove the FAQ, this was something that was highlighted as something we needed in our docs due from user research. I understand it only has one entry currently - maybe a better fix is to expand it? cc @evankanderson